spec/features/workflow/schedule_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.feature "Scheduling an edition" do
+  scenario do
+    given_there_is_an_edition_submitted_for_review
```

Comment:
Can we show scheduling if it's not submitted for review?

Reply:
Yes, if it's in draft.  It feels repetitive to cover both states but let me know what you think.

Reply:
I think the problem from a feature point of view is that we should either explicitly test both or none. At the moment, we're only testing one case - I think it would be fine if we just said 'there is an edition' (that's in draft by default) and wait for the need for more tests to come along :-).

Reply:
Changed to `given_there_is_an_edition`